app/Services/ContractPayload.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use ArkEcosystem\Crypto\Utils\Address;
+
+final class ContractPayload
```

Comment:
candidate for php sdk?

Reply:
Indeedy - I've added it to https://app.clickup.com/t/86dvb6env